bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/dto/clip2/Button.java

```diff
@@ -26,17 +27,17 @@
  */
 @NonNullByDefault
 public class Button {
-    private @NonNullByDefault({}) @SerializedName("last_event") String lastEvent;
+    private @Nullable @SerializedName("last_event") String lastEvent;
 
     /**
      * @return the last button event as an enum.
-     * @throws IllegalArgumentException if lastEvent is bad.
+     * @throws IllegalArgumentException if lastEvent is null or bad.
```

Comment:
Minor: There is no need to declare `IllegalArgumentException`. This is a runtime exception designed to fail fast, and should not normally be handled (but rather avoided).